x-pack/plugins/cases/server/services/cases/index.ts

```diff
@@ -418,7 +300,6 @@ export class CasesService {
       unsecuredSavedObjectsClient,
       options: {
         ...caseOptions,
-        fields: includeFieldsRequiredForAuthentication([caseTypeField]),
```

Comment:
Don't we need that?

Reply:
I'm going to be removing the `type` field from the case saved object so the function call would just return `undefined` if the don't pass anything to it.